Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -71,6 +71,21 @@ private void Context_PropertyChanged(object sender, System.ComponentModel.Proper
                     LoadActionFrame(null); 
                 }                
             }
+            else if(e.PropertyName is nameof(mContext.Environment))
```

Comment:
why to reset all each time user changes the selected environment? users doing it a lot during use of Automate page with same BF/Solution